### ISSUE TITLE
Support for GGUF versions?

### ISSUE THREAD

**Issue by fusillustrator** [kijai/ComfyUI-HunyuanVideoWrapper#192]
I try to connect the model using gguf loader and it doesnot link

**Comment by shaun-ba** [comment]
Share your workflow

**Comment by fusillustrator** [comment]
> Share your workflow.
> 
> If you are trying to connect "Load Diffusion Model" to the Hunyuan node that won't work, you'll have to use Ksampler I think.
> 
> Here is a workflow you can try if you can just about make it out, please do share your results.
> 
> ![2024-12-21_09-38](https://private-user-images.githubusercontent.com/3339587/397918591-d1d9523a-1040-40a2-bde3-69fc24dc722c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ3NzgwNzcsIm5iZiI6MTczNDc3Nzc3NywicGF0aCI6Ii8zMzM5NTg3LzM5NzkxODU5MS1kMWQ5NTIzYS0xMDQwLTQwYTItYmRlMy02OWZjMjRkYzcyMmMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTIyMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEyMjFUMTA0MjU3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9OTQ4NTM4NjAyOWRlMjJjZTYyMTBjNzAzYThjNzgyNjVhODI5MWJlNDI2NDhjMmY3MTMwNzNhMGY3NjQ3ODVmMSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.WygeyA6duAaEKyMUDMxYYcgt6N6p3IC5YdoP9FabLHA)

I wanted to test that gguf on video to video thats why

**Comment by shaun-ba** [comment]
> I wanted to test that gguf on video to video thats why

Read my reply fully.

**Comment by BahamutRU** [comment]
> Here is a workflow

It's just a screen, not workflow. Can you send workflow? =)

**Comment by shaun-ba** [comment]
> It's just a screen, not workflow. Can you send workflow? =)

Don't be lazy create it yourself, I don't need it.

**Comment by pondloso** [comment]
> Share your workflow.
> 
> If you are trying to connect "Load Diffusion Model" to the Hunyuan node that won't work, you'll have to use Ksampler I think.
> 
> Here is a workflow you can try if you can just about make it out, please do share your results.
> 
> ![2024-12-21_09-38](https://private-user-images.githubusercontent.com/3339587/397918591-d1d9523a-1040-40a2-bde3-69fc24dc722c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ4Njk5NzksIm5iZiI6MTczNDg2OTY3OSwicGF0aCI6Ii8zMzM5NTg3LzM5NzkxODU5MS1kMWQ5NTIzYS0xMDQwLTQwYTItYmRlMy02OWZjMjRkYzcyMmMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTIyMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEyMjJUMTIxNDM5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZDAyMGQ1MGY2ZjdiZDJmZDViNWJjZmU3MzBlNzcxNmU5NmY2MmZiNDU1ODI5NzkzMGZlMTk4YmZjZmUxZGE0OSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.PwoLmOgO96D3kZ1QXOVMzKzLdyRH5ll0S6kdy9npwA0)

this is Hunyuan work flow from comfyui core it not from kijai node.

**Comment by shaun-ba** [comment]
Yes I know that... This node doesn't support it directly, I was trying to help someone who clearly isn't even interested in helping themselves, I understand why people here and Kijai himself get fed up with users of their code to be honest.

You all need to actually try and get things working, then report actual bugs, rather than being spoon fed all the time.

**Comment by snapicda** [comment]
https://huggingface.co/Kijai/HunyuanVideo_comfy/blob/main/hyvid_dashtoon_keyframe_native_example_01.json
i find there is already have an example wolkflow, thanks a lot for kijai
